src/context/WalletProvider/KeepKey/components/DownloadUpdater.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Icon, Link, ModalBody, ModalHeader, Text as CText } from '@chakra-ui/react'
+import { FaApple, FaLinux, FaWindows } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { Text } from 'components/Text'
+
+import { getPlatform } from '../helpers'
+
+export const KeepKeyDownloadUpdater = () => {
+  const platform = getPlatform()
+
+  const platformFilename =
+    platform === 'Mac OS'
+      ? 'KeepKey-Updater-2.1.4.dmg'
+      : platform === 'Windows'
+      ? 'KeepKey-Updater-Setup-2.1.4.exe'
+      : 'KeepKey-Updater-2.1.4.AppImage'
+
+  const platformIcon =
+    platform === 'Mac OS' ? FaApple : platform === 'Windows' ? FaWindows : FaLinux
+
+  const updaterUrl = `https://github.com/keepkey/keepkey-updater/releases/download/v2.1.4/${platformFilename}`
+  const releasePage = 'https://github.com/keepkey/keepkey-updater/releases/latest'
+  const translate = useTranslate()
+
+  return (
+    <>
+      <ModalHeader textAlign='center'>
+        <Text translation={'modals.keepKey.downloadUpdater.header'} />
+      </ModalHeader>
+      <ModalBody textAlign='center'>
+        <Icon as={platformIcon} boxSize={20} mb={4} color='white' />
+        <CText fontWeight='bold'>{platform}</CText>
+        <Link isExternal href={releasePage}>
+          <Text
+            color='gray.500'
+            translation={translate('modals.keepKey.downloadUpdater.wrongPlatform', {
```

Comment:
No need to call `translate()` here, that's already done with `translation` prop.
The reason why this works is because we're calling `translate()` twice, once on the original string, then on the translated and interpolated string - since it is not found, we return the "translation key" (which is in fact the translated and interpolated string) as a fallback.